src/consensus/aft/test/logging_stub.h

```diff
@@ -222,6 +222,58 @@ namespace aft
       return kv::DeserialiseSuccess::PASS;
     }
 
+    class ExecutionWrapper : public kv::AbstractExecutionWrapper
+    {
+    public:
+      ExecutionWrapper(const std::vector<uint8_t>& data_) : data(data_) {}
+
+      kv::DeserialiseSuccess Execute() override
```

Comment:
Our naming convention is lower-case, underscore-separated for functions/methods.